### PR TITLE
Feat: Added cart section

### PR DESCRIPTION
**Pull request by abhijeetekad**
Added cart section:
chrome-capture (2)

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  display: flex;
+  justify-content: space-between;
+  width: 70%;
+}
+.product-img {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 60%;
+  padding: 0rem 1rem 0rem 0rem;
```
Comment (ZuberDunge) [inline css/cart.css]: suggestion : you can use padding right here.

**Review by ZuberDunge — APPROVED** [review]
Add line space after each {}, code will look even better, rest looks fine. Good to go from side.

File: css/cart.css [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  color: var(--header-Color);
+  cursor: pointer;
+}
+.cart-btn:hover {
+  opacity: 0.6;
+}
+.price-details {
+  width: 70rem;
+  height: 25rem;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```
Comment (ArjunGTX) [inline css/cart.css]: Shadows can also be stored in variables

**Review by ArjunGTX — APPROVED** [review]
Everything else looks good to me

File: css/cart.css (hunk 1 above)
Comment (abhijeetekad) [reply css/cart.css]: Done

File: css/cart.css (hunk 2 above)
Comment (abhijeetekad) [reply css/cart.css]: Done

**Pull request merged**